mesheryctl/internal/cli/root/root.go

```diff
@@ -141,33 +141,42 @@ func initConfig() {
 	//  perf:
 	//	  authTokenURI:    "/gettoken",
 	//	  loadTestSmpURI: "/perf/load-test-smp",
+
+	defaultsMap := map[string]interface{}{
+		"baseMesheryURL": "http://localhost:9081/api",
+		"perf": map[string]interface{}{
+			"authTokenURI":   "/gettoken",
+			"loadTestSmpURI": "/perf/load-test-smp]",
+			"loadTestURI":    "/perf/load-test",
+		},
+		"ctlversion": map[string]interface{}{
+			"build":     version,
+			"commitsha": commitsha,
+		}}
+
 	if mctlCfgFile != "" {
 		viper.SetConfigFile(mctlCfgFile)
 		if err := viper.ReadInConfig(); err == nil {
 			log.Debugf("Using mesheryctl config file: %s", viper.ConfigFileUsed())
 		} else {
-			log.Fatal(err)
+			log.Info(err)
+			generateMesheryctlConfigUsingDefaultValues(defaultsMap)
```

Comment:
Maybe an Info log alerting the user that a new configuration has been generated?

Reply:
NVM I see that you long within the function. Ignore this